CorsixTH/Lua/dialogs/resizables/hotkey_assign.lua

```diff
@@ -434,297 +410,187 @@ function UIHotkeyAssign:UIHotkeyAssign(ui, mode)
+    {
+      id = "toggle",
+      title = _S.hotkey_window.panel_toggleKeys,
+      tooltip = _S.hotkey_window.panel_toggleKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_toggleKeys,
+          keys = {
+            "ingame_toggleAnnouncements",
+            "ingame_toggleSounds",
+            "ingame_toggleMusic",
+            "ingame_toggleAdvisor",
+            "ingame_toggleInfo" }}}},
+    {
+      id = "panels",
+      title = _S.hotkey_window.caption_panels,
+      tooltip = _S.tooltip.hotkey_window.caption_panels,
+      sections = {{
+          title = _S.hotkey_window.caption_panels,
+          keys = {
+            "ingame_panel_bankManager",
+            "ingame_panel_bankStats",
+            "ingame_panel_staffManage",
+            "ingame_panel_townMap",
+            "ingame_panel_casebook",
+            "ingame_panel_research",
+            "ingame_panel_status",
+            "ingame_panel_charts",
+            "ingame_panel_policy",
+            "ingame_panel_buildRoom",
+            "ingame_panel_furnishCorridor",
+            "ingame_panel_editRoom",
+            "ingame_panel_hireStaff",
+            "ingame_panel_map_alt",
+            "ingame_panel_research_alt",
+            "ingame_panel_casebook_alt",
+            "ingame_panel_casebook_alt02" }}}},
+    {
+      id = "recall",
+      title = _S.hotkey_window.button_recallPosKeys,
+      tooltip = _S.tooltip.hotkey_window.button_recallPosKeys,
+      sections = {{
+          title = _S.hotkey_window.panel_storePosKeys,
+          keys = {
+            "ingame_storePosition_1",
+            "ingame_storePosition_2",
+            "ingame_storePosition_3",
+            "ingame_storePosition_4",
+            "ingame_storePosition_5",
+            "ingame_storePosition_6",
+            "ingame_storePosition_7",
+            "ingame_storePosition_8",
+            "ingame_storePosition_9",
+            "ingame_storePosition_0" }},
+        {
+          title = _S.hotkey_window.panel_recallPosKeys,
+          keys = {
+            "ingame_recallPosition_1",
+            "ingame_recallPosition_2",
+            "ingame_recallPosition_3",
+            "ingame_recallPosition_4",
+            "ingame_recallPosition_5",
+            "ingame_recallPosition_6",
+            "ingame_recallPosition_7",
+            "ingame_recallPosition_8",
+            "ingame_recallPosition_9",
+            "ingame_recallPosition_0" }}}},
+  }
+
+  if self.ui.app.config.debug then
+    table.insert(self.key_options, {
+      id = "debug",
+      title = _S.hotkey_window.panel_debugKeys,
+      tooltip = _S.hotkey_window.panel_debugKeys,
+      sections = {{
+        title = _S.hotkey_window.panel_debugKeys,
+        keys = {
+          "global_connectDebugger",
+          "global_showLuaConsole",
+          "global_runDebugScript",
+          "ingame_showCheatWindow",
+          "ingame_poopLog",
+          "ingame_poopStrings" }}}})
+  end
+
```

Comment:
You've got indentation inconsistencies here. Some are 2, some are 4.

Reply:
Fixed